src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -233,7 +252,7 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
       })
     }
 
-    if (!isClickableOutsideFocusTrap) {
+    if (!isClickableOutsideFocusTrap || !focusTriggerOnOutsideClick) {
```

Comment:
it is better to encapsulate this condition under single expression that will be used in both places, like

```js
private shouldHandleOutsideClick = () => 
  !this.props.isClickableOutsideFocusTrap || !this.props.focusTriggerOnOutsideClick

if (this.shouldHandleOutsideClick()) {
  ...
}
```

This move will ensure consistency of this policy applied at several places.

Reply:
make sense, will change